be/src/olap/tablet_meta_manager.cpp

```diff
@@ -78,9 +76,9 @@ OLAPStatus TabletMetaManager::get_json_header(DataDir* store,
 }
 
 OLAPStatus TabletMetaManager::save(DataDir* store,
-        TTabletId tablet_id, TSchemaHash schema_hash, const TabletMeta* tablet_meta) {
+        TTabletId tablet_id, TSchemaHash schema_hash, const TabletMeta* tablet_meta, string header_prefix) {
```

Comment:
I think header_prefxi should be renamed to meta_prefix or just prefix

Reply:
it should be header prefix because the const variable's name is header prefix and meta is too broad.